### ISSUE TITLE
Regarding the data division of memotion7k

### ISSUE THREAD

**Issue by PotatoDog1669** [valeriolagatta/KERMIT_MemeDetection#3]
How is the training and test set split in memotion7k?

**Comment by valeriolagatta** [comment]
Thanks for your interest in this research. 

The training/test split was performed in a stratified way using `sklearn.model_selection.train_test_split` with random seed set to 42. The partitions ratio is 80:10:10 for training, validation, test sets, respectively.

You can refer to the original dataset ([memotion7k](https://www.kaggle.com/datasets/williamscott701/memotion-dataset-7k)), particularly the file `labels.csv`
